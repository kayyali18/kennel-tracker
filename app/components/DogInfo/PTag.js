import styled from 'styled-components'

const PTag = styled.div`
  border-right: 2px solid grey;
  font-size: 20px;
  border-left: 2px solid grey;
  margin: 0 auto;
  max-width: 500px;
  padding: 10px;
  text-align: center;
  color: grey;
  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
  color: #595959;
`
export default PTag
