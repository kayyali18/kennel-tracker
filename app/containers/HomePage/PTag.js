import styled from 'styled-components'

const PTag = styled.div`
  text-align: center;
  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
  color: #595959;
`
export default PTag
