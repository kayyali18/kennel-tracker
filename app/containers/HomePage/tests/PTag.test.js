import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import PTag from '../PTag'

describe('<PTag />', () => {
  it('should match snapshot', () => {
    const renderedComponent = renderer.create(<PTag />).toJSON()
    expect(renderedComponent).toMatchSnapshot()
  })

  it('should render a p tag', () => {
    const renderedComponent = renderer.create(<PTag />).toJSON()
    expect(renderedComponent.type).toEqual('p')
  })
})
