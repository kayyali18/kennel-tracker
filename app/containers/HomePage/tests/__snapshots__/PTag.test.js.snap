// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PTag /> should match snapshot 1`] = `
.c0 {
  text-align: center;
  font-family: 'Helvetica Neue',Helvetica,Arial,sans-serif;
  color: black;
}

<p
  className="c0"
/>
`;
